src/components/infotext/index.js

```diff
@@ -0,0 +1,132 @@
+import { nativeVlElement, define } from '../../utils/core';
```

Comment:
Ik krijg de volgende error bij gebruik van de component; 
```
infotext.js:147 Uncaught Error: Set a static height and width on the target element <div data-vl-value="" class="vl-infotext__value" data-vl-infotext-value="" data-vl-js-dress="false" data-vl-infotext-dressed="true" style="white-space: nowrap;"><span class="textFitted" style="display: inline-block; font-size: 38px;">
        <span style="font-size: 1.25em;">150</span>
        <span style="font-size: 1em;">cm</span>
      </span></div> before using textFit!
    at processItem (infotext.js:147)
    at textFit (infotext.js:120)
    at Infotext.dress (infotext.js:288)
    at HTMLDivElement.__dress (index.js:110)
    at HTMLDivElement.__processChildElementResize (index.js:106)
    at ResizeObserver.<anonymous> (index.js:31)

Reply:
Ik heb deze net geprobeerd in onze react-test app en daar werkt hij zonder problemen.